Packs/Bonusly/Scripts/IncOwnerToBonuslyUser.yml

```diff
@@ -0,0 +1,52 @@
+commonfields:
+  id: d340ca1a-8573-4f9d-86cb-fafb7f5d7ea2
+  version: 7
+name: IncOwnerToBonuslyUser
+script: |-
+  """
+  Pass in JSON key value lookup then fetches the incident owner and finds the bonusly user
+
+  """
+  import json
+
+  json_lookup = demisto.args().get('json')
+
+
+  def inc_owner_bonusly_user():
+      owner_username = demisto.incidents()[0].get("owner")
```

Comment:
same issue as above - I am not sure this is what you wanted to achieve - please explain.

Reply:
Both of these are used to get the owner of the incident's email address and utilized in the playbook as the owner in the product is only the username and not their email. For their playbook to work, you need the incident owners email.

Reply:
same as above  - please create an argument for owner_username and remove this line. Use the incident context to fill this argument in the playbook.